assets/scss/6-components/mini-form/mini-form.html

```diff
@@ -0,0 +1,16 @@
+<form class="c-mini-form">
+  <div class="c-mini-form__input">
+    <input id="email" name="email" aria-label="email address to link" type="text" class="c-text-input is-invalid l-display-block l-width-full has-text-gray-dark has-xxxs-btm-marg">
+    <ul>
+      <li class="has-text-error has-xxxs-btm-marg t-size-xs">
+        First validation error
```

Comment:
@AndrewGibson27 @ashley-hebler: Food for thought ... should the verification error message appear between the label and the input? I'm imagining a situation where there are several form fields in a row ... making sure people know which one has the error. Thoughts?

Reply:
I think we can more cleanly handle that with good spacing. But can adjust if it becomes a problem.